inc/saiswitch.h

```diff
@@ -2265,6 +2394,16 @@ typedef enum _sai_switch_attr_t
      */
     SAI_SWITCH_ATTR_TUNNEL_LOOPBACK_PACKET_ACTION,
 
+    /**
+     * @brief Switch scoped Tunnel objects
+     *
+     * @type sai_object_list_t
+     * @flags CREATE_AND_SET
+     * @objects SAI_OBJECT_TYPE_SWITCH_TUNNEL
+     * @default empty
+     */
+    SAI_SWITCH_ATTR_TUNNEL_OBJECTS_LIST,
```

Comment:
Can we have this as read only and have switch object ID as attribute in the sai_switch_tunnel_object? This will make it inline with other objects

Reply:
yes, switch_id is passed when create object function is called, so you already have it.
i don't see a point to have object_type_switch on any attribute, and even if you want to get switch id from tunnel object id, then use sai_switch_id_query()

Reply:
@kcudnik Sudharsan was thinking if we can create a binding of switch object in SAI_OBJECT_TYPE_SWITCH_TUNNEL. I think thats implicit.
@dgsudharsan Sudharsan, what do you think?

Reply:
yes, thats implicit